activesupport/lib/active_support/logger.rb

```diff
@@ -14,9 +14,16 @@ class Logger < ::Logger
     #   ActiveSupport::Logger.logger_outputs_to?(logger, STDOUT)
     #   # => true
     def self.logger_outputs_to?(logger, *sources)
```

Comment:
This method was not working properly even before. It was only checking if the main logger (the broadcast one) would output to `sources` but didn't consider that other loggers part of the broadcast could have.